src/modules/mount/main.py

```diff
@@ -80,37 +84,44 @@ def mount_partitions(root_mount_point, partitions):
             subprocess.check_call(["umount", "-v", root_mount_point])
 
             if "luksMapperName" in partition:
-                libcalamares.utils.mount("/dev/mapper/{!s}".format(partition["luksMapperName"]),
+                libcalamares.utils.mount("/dev/mapper/"
+                                         + partition["luksMapperName"],
                                          mount_point,
                                          fstype,
                                          ",".join(["subvol=@",
-                                                   partition.get("options", "")]),
+                                                   partition.get("options",
+                                                                 "")]),
```

Comment:
Here, too (and elsewhere in this PR), shifting all the arguments down one line and using one indent, as opposed to visual indent, works and IMO improves readability by keeping  the `get()` together.